src/Chart.Core.js

```diff
@@ -351,7 +351,7 @@
 				return 0;
 			}
 		},
-		toRadians = helpers.radians = function(degrees){
+		toRadians = helpers.toRadians = function(degrees){
```

Comment:
No one used this function from the helpers class. All usages were to the toRadians var. I aligned the function name on the helpers class.